src/runners/violation_suppressions_runner.py

```diff
@@ -24,17 +31,44 @@ def run_suppression(squelch_name):
             SET t.suppressed='true', t.suppression_rule='{squelch_name}';
         """)
     except Exception as e:
+        log.metadata_fill(metadata, status='failure', rows=0)
+        RUN_METADATA['QUERY_HISTORY'].append(metadata)
         log.fatal("Suppression query {squelch_name} execution failed.", e)
+        pass
 
     print(f"Suppression query {squelch_name} executed")
+    log.metadata_fill(metadata, status='success', rows=ctx.cursor().rowcount)
+    RUN_METADATA['QUERY_HISTORY'].append(metadata)
+
+
+def record_metadata(ctx, metadata):
```

Comment:
are these identical in the 4 files? can we put them in a helper?

Reply:
record_metadata is slightly different between suppressions and queries, so we can't put it in a helper quite yet. When we get more robust failure logging for suppressions, that becomes a reasonable refactor.